.github/workflows/build.yml

```diff
@@ -16,6 +16,10 @@ jobs:
     name: Test
     needs: gradleValidation
     runs-on: ubuntu-latest
+    env:
+      USERNAME_GITHUB: ${{ github.actor }}
+      READ_TOKEN_GITHUB: ${{ github.token }}
+      DEEPCODE_API_KEY: ${{ secrets.DEEPCODE_API_KEY }}
```

Comment:
Why do we need Deepcode api key?

Reply:
looks like it was used before when (hardcoded) deepcode token was needed to init the plugin
probably could be removed (together with github tokens as well, used for java-client library resolving from guthub afaik) 
but you better check, you know better how that magic works :)

Reply:
did remove that, hopefully nothing will be broken :)